DEFINE-INFERENCE-MODEL.md

```diff
@@ -0,0 +1,283 @@
+# How to define a custom inference model 
+
+Note: See overview at [README](https://github.com/datarobot/datarobot-user-models#readme)
+
+## Content
+
+1. [Assembling an inference model code folder](#inference_model_folder)
+2. [Define structured inference model](#structured_inference_model)
+3. [Define unstructured inference model](#unstructured_inference_model)
+4. [Test an inference model with DRUM locally](#test_inference_model_drum)
+5. [Building your own environment](#build_own_environment)
+6. [Upload, manage, and deploy a custom inference model with DataRobot](#custom_model_runner)
```

Comment:
should we add a link to DR docs here? we actually have a page that describes how to upload, manage, etc custom models: https://app.datarobot.com/docs/mlops/deployment/custom-models/custom-inf-model.html

Reply:
agreed! Added